src/main/ui/app/sass/addonindex.scss

```diff
@@ -26,14 +26,32 @@
   float: right;
 
 }
+.maintable{
+margin-bottom:0;
+}
+
+
+.titletype{
 
+margin-right:5%;
+margin-bottom:5px;
+
+}
+.titlename{
+margin-left:-15%;
+margin-bottom:5px;
+
+}
+.mainlist{
+border-width: 0;
+}
 .pushdown {
   margin-top: 40px;
 }
 
 .textdec {
```

Comment:
What is textdec supposed to mean, anyway? It's not an abbreviation I'm familiar with. In general I'd prefer css class names to be dash-separated, e.g. "main-list" instead of "mainlist"